community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/slurmsync.py

```diff
@@ -70,6 +73,10 @@
     ),
 )
 
+SLURM_CREATE_RESERVATION = "scontrol create reservation user=root starttime={} duration=180 nodes={} reservationname={}"
+SLURM_DELETE_RESERVATION = "scontrol delete reservation {}"
+SLURM_SHOW_RESERVATION = "scontrol show reservation"
```

Comment:
It only used once and never outside, no reason to make it into constant, especially "public" one. Just use string literal where you need it.